README.md

```diff
@@ -43,35 +48,57 @@ Saving the document will move the file from the inbox folder into the outbox fol
 4. Config - Directory where the conf.json file of user configuration options is stored.
 
 ## :arrow_forward: Running the Application
+
 Whilst you could build this project and drop the files into an IIS application, the simplest method is using Docker.
 
 ### :whale2: Docker
 
 1. Pull the image:
+
 ```
 docker pull lbrowning/pdf-web-edit
 ```
-2. Run a container (replacing the host paths with your desired paths): 
-    
+
+2. Run a container (replacing the host paths with your desired paths):
+
 ```
 docker run -p 10000:80 -v /host/config:/config -v /host/inbox:/inbox -v /host/outbox:/outbox -v /host/archive:/archive lbrowning/pdf-web-edit:latest
 ```
 
 Congratulations, PDF Web Edit should now be running on port 10000!
 
+Add it to [ngx-paperless](https://github.com/paperless-ngx/paperless-ngx) `docker-compose.yml`:
+
+```yml
+pdfedit:
+  image: ghcr.io/danielhabenicht/danielhabenicht/fork.pdf-web-edit:v0.9.2
```

Comment:
Any way we can read this from a variable?

Reply:
I don't think so. But if you add something like [mkdocs](https://squidfunk.github.io/mkdocs-material/) and host it as github page you could autogenerate it.

Reply:
Or write a custom action that automatically commits after every release.